src/rules/__tests__/prefer-mocked.test.ts

```diff
@@ -0,0 +1,321 @@
+import path from 'path';
+import dedent from 'dedent';
+import rule from '../prefer-mocked';
+import { FlatCompatRuleTester as RuleTester } from './test-utils';
+
+function getFixturesRootDir(): string {
+  return path.join(__dirname, 'fixtures');
+}
+
+const rootPath = getFixturesRootDir();
+
+const ruleTester = new RuleTester({
+  parser: require.resolve('@typescript-eslint/parser'),
+  parserOptions: {
+    sourceType: 'module',
+    tsconfigRootDir: rootPath,
+    project: './tsconfig.json',
+  },
+});
```

Comment:
you shouldn't need `project` and co since we're not using type information

```suggestion
const ruleTester = new RuleTester({
  parser: require.resolve('@typescript-eslint/parser'),
});
```